packaging/src/docker/README.md

```diff
@@ -210,3 +210,54 @@ docker compose down
     select count(distinct a) from hive_example;
     select sum(b) from hive_example;
   ```
+
+#### Hive Schema Tool
+
+`Hive Schema Tool` is located in the Docker Image at `/opt/hive/bin/schematool`.
+
+By default, system libraries such as `information_schema` for HiveServer2 are not created. 
+To create system libraries for a HiveServer2 instance, 
+users need to configure HiveServer2 to use a remote Hive Metastore Server and use a database other than embedded Derby for the Hive Metastore Server.
+
+Assuming `SDKMAN!` and `Docker CE` are installed, a possible use case is as follows.
+Create a `compose.yaml` file in the current directory,
+
+```yaml
+services:
+  some-postgres:
+    image: postgres:17.2-bookworm
+    environment:
+      POSTGRES_PASSWORD: "example"
+  metastore-standalone:
+    image: apache/hive:4.0.1
+    depends_on:
+      - some-postgres
+    environment:
+      SERVICE_NAME: metastore
+      DB_DRIVER: postgres
+      SERVICE_OPTS: >-
+        -Djavax.jdo.option.ConnectionDriverName=org.postgresql.Driver
+        -Djavax.jdo.option.ConnectionURL=jdbc:postgresql://some-postgres:5432/postgres
+        -Djavax.jdo.option.ConnectionUserName=postgres
+        -Djavax.jdo.option.ConnectionPassword=example
+    volumes:
+      - ~/.m2/repository/org/postgresql/postgresql/42.7.5/postgresql-42.7.5.jar:/opt/hive/lib/postgres.jar
+  hiveserver2-standalone:
+    image: apache/hive:4.0.1
+    depends_on:
+      - metastore-standalone
+    environment:
+      SERVICE_NAME: hiveserver2
+      SERVICE_OPTS: >-
+        -Dhive.metastore.uris=thrift://metastore-standalone:9083
```

Comment:
we need to push the ` -Djavax.jdo.option.ConnectionDriverName=org.postgresql.Driver
        -Djavax.jdo.option.ConnectionURL=jdbc:postgresql://some-postgres:5432/postgres
        -Djavax.jdo.option.ConnectionUserName=postgres
        -Djavax.jdo.option.ConnectionPassword=example` to the HS2 pod as well

Reply:
- There is a new issue at https://github.com/apache/hive/pull/5629#pullrequestreview-2603571362 I will say that I am a bit confused as to why HiveServer2 needs to connect to the Postgres instance used by the Hive Metastore Server....

Reply:
do you post the command the HS2 trying to initialize the schema, it could find at the container's log. 
this is the sys table, the actual data is stored at the postgres, HMS doesn't provide APIs to scan through postgres tables, like `select * from postgres."TBLS"` , the job is carried out by HS2, which connects to postgres and pushes the query down to the postgres.

Reply:
> do you post the command the HS2 trying to initialize the schema, it could find at the container's log. 
> this is the sys table, the actual data is stored at the postgres, HMS doesn't provide APIs to scan through postgres tables, like `select * from postgres."TBLS"` , the job is carried out by HS2, which connects to postgres and pushes the query down to the postgres.

- I have confirmed that the current issue is resolved at https://github.com/apache/hive/pull/5629#pullrequestreview-2603875428. The current PR is ready for further review.